pkgs/development/tools/skopeo/default.nix

```diff
@@ -31,8 +31,7 @@ buildGoModule rec {
 
   nativeBuildInputs = [ pkg-config go-md2man installShellFiles makeWrapper ];
 
-  buildInputs = [ gpgme ]
-  ++ lib.optionals stdenv.isLinux [ lvm2 btrfs-progs ];
+  buildInputs = [ gpgme ] ++ lib.optionals stdenv.isLinux [ lvm2 btrfs-progs ];
```

Comment:
Why is this included with the version bump?

Reply:
This was the auto formatter, I reverted that change.

Reply:
Are you using `nixpkgs-fmt`? Asking about this because we might start using it treewide soon and it shouldn't change multi-line into single-line.

Reply:
Yes, that's exactly how it behaved on my side too. I did the line-merge manually since it was still lower than 80 chars width

Reply:
Ah okay, sorry for the noise.